hack/jenkins/e2e-runner.sh

```diff
@@ -232,7 +232,11 @@ fi
 # Install gcloud from a custom path if provided. Used to test GKE with gcloud
 # at HEAD, release candidate.
 if [[ ! -z "${CLOUDSDK_BUCKET:-}" ]]; then
-    gsutil -mq cp -r "${CLOUDSDK_BUCKET}" ~
+    # Retry the download a few times to mitigate transient server errors and
+    # race conditions where the bucket contents change under us as we download.
+    for n in $(seq 3); do
+        gsutil -mq cp -r "${CLOUDSDK_BUCKET}" ~ && break || sleep 1
```

Comment:
Does this actually restart the copy from the beginning? I happen to recall some state file that was local and could resume a cp. If the contents changes as we were cp'ing what's the desired behavior?